modules/gdscript/gdscript_parser.cpp

```diff
@@ -1060,8 +1060,8 @@ void GDScriptParser::parse_class_body(bool p_is_multiline) {
 			default:
 				// Display a completion with identifiers.
 				make_completion_context(COMPLETION_IDENTIFIER, nullptr);
-				push_error(vformat(R"(Unexpected "%s" in class body.)", current.get_name()));
 				advance();
+				push_error(vformat(R"(Unexpected "%s" in class body.)", previous.source));
```

Comment:
Using the token source code in the error message is debatable, since tokens can be multiline or unreadable (`LITERAL`, `NEWLINE`, `INDENT`, `DEDENT`). I think we should use something like:

```c++
const String token_name = previous.is_identifier() ? previous.get_identifier() : previous.get_name();
```

Or maybe add a new method to `GDScriptTokenizer::Token`.

Reply:
Updated